takeshape/takeshape-starter-api-indexing-auth0-stripe
# Add API Indexing to Auth0+Stripe Starter

Adds API Indexing to the Auth0+Stripe starter

### Test Plan (Steps to test):

1. Deploy the pattern
1. Clone the repo and pull this branch
1. Follow the README instructions
1. Run the demo
1. Ensure the demo runs smoothly
